docs/src/main/sphinx/release/release-424.md

```diff
@@ -0,0 +1,46 @@
+# Release 424 (16 Aug 2023)
+
+## General
+
+* Improve query performance for large Trino clusters with many worker nodes. ({issue}`18542`)
+* Require the JVM default charset to be UTF-8. This can be set with the JVM
+  command line option `-Dfile.encoding=UTF-8`. ({issue}`18657`)
+
+## JDBC driver
+
+* Add the number of bytes written to HTTP to the query results response. ({issue}`18651`)
+
+## Delta Lake connector
+
+* Remove the legacy Parquet reader, along with the
+  `parquet.optimized-reader.enabled` and
+  `parquet.optimized-nested-reader.enabled` configuration properties. ({issue}`18639`)
+
+## Hive connector
+
+* Remove the legacy Parquet reader, along with the
+  `parquet.optimized-reader.enabled` and
+  `parquet.optimized-nested-reader.enabled` configuration properties. ({issue}`18639`)
+
+## Hudi connector
+
+* Remove the legacy Parquet reader, along with the
+  `parquet.optimized-reader.enabled` and
+  `parquet.optimized-nested-reader.enabled` configuration properties. ({issue}`18639`)
+
+## Iceberg connector
+
+* Add support for `CASCADE` option in `DROP SCHEMA` statements. ({issue}`18689`)
+* Remove the legacy Parquet reader, along with the
+  `parquet.optimized-reader.enabled` and
+  `parquet.optimized-nested-reader.enabled` configuration properties. ({issue}`18639`)
+* Fix potential incorrect query results when a query involves a predicate on a
+  `timestamp with time zone` column. ({issue}`18588`)
+
+## Memory connector
+
+* Add support for `CASCADE` option in `DROP SCHEMA` statement. ({issue}`18668`)
```

Comment:
```suggestion
* Add support for `CASCADE` option with `DROP SCHEMA`. ({issue}`18668`)
```

Reply:
The original phrasing sounds better.

Reply:
Okay.. we just did similar changes recently. I like the original one more as well.